app/controllers/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def index
+    @users = User.order(created_at: :desc)
+  end
+
+  def create
+    @user = User.new(create_params)
+    if @user.save
+      flash[:info] = "#{@user.email} has been created with the password of #{create_params[:password]}"
+      redirect_to users_path
+    else
+      render :new
```

Comment:
No flash message if there's an error? I have a feeling devise doe something clever with the model.errors here, but can't remember